2.adoc

```diff
@@ -72,5 +72,229 @@ Optional description
 . Minus A
 . Minus B
 
+=== GitBook
+
+Desktopový (free) a online (placený) editor pro markupové dokumenty ve formátu GFM (GitHub Flavored Markdown) a AsciiDoc.
+
+Navíc k editaci dokumentů umožňuje správu celých knih. Resp. používá metaforu knihy na rozdělení projektu do kapitol, případně do souborů.
+
+Na pozadí GitBook verzuje dokumenty pomocí gitu. Prohlížení změn dokumentu je tak pohled nad git commity.
+
+link:+https://www.gitbook.com/editor+[GitBook]
+
+==
+
+Klikání do preview neodscrolluje markup zdroj do správné pozice, což může být nepříjemné obzvláště u delších dokumentů.
+
+==== Porovnání rozdílů
+
+. GitBook je stand-alone produkt - jak desktopová, tak webová verze. Nelze jej tudíž embedovat do jiných stránek.
+. Neobsahuje command palette, místo toho má nástrojovou lištu pro vkládání základních markupových konstruktů.
+* GitBook umožňuje kód editovat nejen v source+preview módu, ale i jako WYSIWYG.
+. Obsahuje dialog pro vkládání obrázků, které lze načítat z lokální knihovny, disku, či vzdálené adresy
+
+==== Zhodnocení UI
+
+===== Základní pohled
+
+image:/assets/zakladni-ui.png[]
+
+Základní UI zobrazuje ve své horní části nástrojové menu, konfiguraci verzí a markdown paletu.
```

Comment:
V ostatnich resersich nastrojove menu = tool bar, ale to je asi detail..

Reply:
opraveno